environ/src/environ/util.cljc

```diff
@@ -0,0 +1,8 @@
+(ns environ.util
```

Comment:
Let's not add a new namespace. Instead create a `core.cljc`, and just use reader conditionals to separate things out.